tests/util/models/gpflux/models.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import gpflow
+import tensorflow as tf
+from gpflux.layers import GPLayer
+from gpflux.models import DeepGP
+
+from trieste.data import TensorType
+
+tf.keras.backend.set_floatx("float64")
```

Comment:
A little bit worries  about side effects of this call. That will affect all files which, even indirectly, import this one, right? Perhaps it's safer to run this line where the tests are? Or does it have to be here for some reason?

Reply:
No I think you're right, I'll move this

Reply:
I think this has been accidentally left in